drives.go

```diff
@@ -103,3 +103,15 @@ func WithRateLimiter(limiter models.RateLimiter) DriveOpt {
 		d.RateLimiter = &limiter
 	}
 }
+
+// WithCacheType sets the cache strategy for the block device
+func WithCacheType(cache_type string) DriveOpt {
+	if cache_type == "" {
+		return func(_ *models.Drive) {
+			// no-op
+		}
+	}
```

Comment:
Seems we don't do this kind of check in most of helpers here. Do you have specific reasons to do here?

One concern I have is that if someone pass `""` unintentionally, it would be ignored. Should we pass `""` as is and let Firecracker error out?

Reply:
I do see your point about letting Firecracker handle everything as is. I added this input validation since I might have to use this later from FC-CD and pass `""`. Though I can add a helper in FC-CD to do just that. 

Will remove this check from here.